GUI wrapper for windows.

jobitjoseph opened the pull request:
Added python based GUI wraper for windows. Make sure python is installed. Jyst run the Image2cGUI.py. (dependencies : tkinter , Will propt to install if not already installed, will automatically install using pip).

jobitjoseph committed:
Add files via upload

jobitjoseph committed:
Update README.md

jobitjoseph committed:
Update README.md

bitbank2 commented:
This change I find quite interesting. I'm not a python person, but I can see the appeal. It revealed a bug in the --strip option, that I need to fix. Besides that, I had to slice and dice the script to run on MacOS. I think the point of python is to be portable and your script as-is is definitely not portable. You've got Windows specific references and 32 vs 64-bit executable tests too. If you can modify it to be non-windows specific (it is possible), then I'll merge it.
Thanks

jobitjoseph committed:
Add files via upload

jobitjoseph committed:
Update main.c

jobitjoseph committed:
Delete Image2cGUI.py

jobitjoseph committed:
Add files via upload

jobitjoseph committed:
Add files via upload

jobitjoseph committed:
Update README.md

jobitjoseph committed:
Merge branch 'bitbank2:master' into master

jobitjoseph committed:
Add files via upload

jobitjoseph commented:
> This change I find quite interesting. I'm not a python person, but I can see the appeal. It revealed a bug in the --strip option, that I need to fix. Besides that, I had to slice and dice the script to run on MacOS. I think the point of python is to be portable and your script as-is is definitely not portable. You've got Windows specific references and 32 vs 64-bit executable tests too. If you can modify it to be non-windows specific (it is possible), then I'll merge it. Thanks

Hi , I have modified the code to work with both windows and MAC and created standalone binary. No more dependencies. binary can be directly run from the corresponding folder. Added binary for the MAC OS too. Havn't tested it since I don't own a MAC. Kindly please check and merge if possible. Thank you
